index.html

```diff
@@ -11,14 +11,237 @@
         <link rel="apple-touch-icon" href="icon.png">
         <!-- Place favicon.ico in the root directory -->
 
+        <link rel="stylesheet" href="node_modules/slick-carousel/slick/slick.css">
+        <link rel="stylesheet" href="node_modules/slick-carousel/slick/slick-theme.css">
         <link rel="stylesheet" href="assets/css/styles.css">
     </head>
     <body>
         
+        <header class="header">
+            <div>
+                <img src="assets/img/goodcall.svg" alt="" class="header__logo">
+            </div>
+            <div class="header__items">
+                <a href="#" data-reveal-id="myModal" class="header__login">Login</a>
+                <img src="assets/img/burger.svg" alt="" class="header__burger">
+            </div>
+        </header>
+
+        <section class="section section--hero  text-center">
+            <div class="row">
+                <div class="columns large-10 large-offset-1">
+                    <div class="slider">
+                        <div class="slide">
+                            <img src="assets/img/slider1.png" alt="" class="slide__img1">
```

Comment:
If your class is going to be `.slide__img1`, I would consider putting the `.slide__img` class on all of them, and then adding the modifier class of `.slide__img--1`.